pkg/scheduler/plugins/loadaware/service.go

```diff
@@ -30,6 +32,14 @@ var _ services.APIServiceProvider = &Plugin{}
 
 type NodeAssignInfoData struct {
 	Pods []PodAssignInfoData `json:"pods,omitempty"`
+
```

Comment:
Same as above, hope to provide user documentation here: https://koordinator.sh/docs/user-manuals/load-aware-scheduling